Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -106,12 +106,25 @@ public struct AppleFrameworkImport: BazelTarget {
     // )
     public func toSkylark() -> SkylarkNode {
         let isDynamicFramework = GetBuildOptions().isDynamicFramework
-
+        let isXCFramework = GetBuildOptions().isXCFramework
+        
+        let frameworkType : String
+        
+        if isXCFramework && isDynamicFramework {
+            frameworkType = "apple_dynamic_xcframework_import"
+        } else if isXCFramework && !isDynamicFramework {
+            frameworkType = "apple_static_xcframework_import"
+        } else if !isXCFramework && isDynamicFramework {
+            frameworkType = "apple_dynamic_framework_import"
+        } else {
+            frameworkType = "apple_static_framework_import"
+        }
```

Comment:
Could this logic be extracted to somewhere common and also used in the `makeLoadNodes`?

Reply:
good idea! simplified in next commit.